code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -90,6 +93,60 @@
 		user.filters = null
 		animation_playing = FALSE
 
+/obj/item/borg_chameleon/proc/fulp_borg_chameleon_menu(mob/living/silicon/robot/user) //Surrealaser's code
+	if(!user)
+		return FALSE
+
+	var/list/borg_disguses_list = list(
+	"Clown", \
+	"Engineering", \
+	"Janitor", \
+	"Medical", \
+	"Mining", \
+	"Peacekeeper", \
+	"Security", \
+	"Service", \
+	"Standard", \
+	"Cancel" = null )
+
+	var/choice = input(user,"Which cyborg module will you disguise as?","Chameleon Cyborg Disguise") as null|anything in borg_disguses_list
+	if(QDELETED(src) || user.stat || !in_range(user, src) || user.incapacitated() || !choice)
+		return FALSE
+
+	switch(choice)
+		if("Medical")
+			disguise_text = "medical"
+			disguise = "medical"
+		if("Security")
+			disguise_text = "security"
+			disguise = "sec"
+		if("Engineering")
+			disguise_text = "engineering"
+			disguise = "engineer"
+		if("Peacekeeper")
+			disguise_text = "peacekeeper"
+			disguise = "peace"
+		if("Janitor")
+			disguise_text = "janitor"
+			disguise = "janitor"
+		if("Clown")
+			disguise_text = "clown"
+			disguise = "clown"
+		if("Mining")
+			disguise_text = "miner"
+			disguise = pick("miner","minerOLD","spidermin")
+		if("Service")
+			disguise_text = "service"
+			disguise = pick("service_f", "service_m", "brobot", "kent", "tophat")
+		if("Standard")
+			disguise_text = "standard"
+			disguise = pick("robot")
+		if("Cancel")
+			return FALSE
```

Comment:
```suggestion
	switch(choice)
		if("Security")
			disguise_text = "security"
			disguise = "sec"
		if("Engineering")
			disguise_text = "engineering"
			disguise = "engineer"
		if("Peacekeeper")
			disguise_text = "peacekeeper"
			disguise = "peace"
		if("Mining")
			disguise_text = "miner"
			disguise = pick("miner","minerOLD","spidermin")
		if("Service")
			disguise_text = "service"
			disguise = pick("service_f", "service_m", "brobot", "kent", "tophat")
		if("Standard")
			disguise_text = "standard"
			disguise = pick("robot")
		if("Cancel")
			return FALSE
		else
			disguise_text = disguise = lowertext(choice)
```
Might want to change some `disguise` names to match the `disguise_text` if there's no reason not to.